writethedocs-na-2016/videos/7-values-of-effective-tech-writing-teams.json

```diff
@@ -0,0 +1,15 @@
+{
+  "description": "Joao Fernandes\nhttp://lanyrd.com/2016/writethedocs/sfbztk/\nNot so long ago, designers were not part of the product development cycle. They were only called at the last minute, with the hopes they could help salvage the project. But by then they couldn&#39;t do much, since it&#39;s hard to backtrack from bad design decisions made early on.\r\nFortunately this has changed. Both management and developers now see the value of including designers as part of the development cycle.\r\nThe same has to happen with tech writers!\r\nIn this talk I&#39;ll share with you the 7 core values my tech writing team used at OutSystems, where we changed the company culture and won our place in the development cycle.\r\nThese values are also being useful to me at Docker, where I&#39;m still learning about the product, the users, and the best way to ship documentation.",
+  "recorded": "2013-05-22",
```

Comment:
I'm not interested in going through each talk and figuring out what day it was given on. Thus, I'm defaulting to the first day of the conference.